grakn-graql/src/test/java/ai/grakn/graql/internal/parser/QueryParserTest.java

```diff
@@ -645,9 +646,10 @@ public void testParseComputeClusterWithSizeThenMembersThenSize() {
         assertEquals(expected, parsed);
     }
 
+    @Ignore
     @Test
     public void testParseComputeDegree() {
-        assertParseEquivalence("compute degrees in movie;");
+        assertParseEquivalence("compute centrality in movie; using degree;");
     }
```

Comment:
Is `compute degrees in movie;` still supported? If it is, we can just leave this test as it is and not ignore it.

Otherwise, we should add a `TODO` to say we have to wait until we support it in the parser.

(same for the other tests below)

Reply:
We plan to remove them eventually, for now I'll make them deprecated and add a `TODO` as well.